packages/bundle-source/src/index.js

```diff
@@ -9,13 +9,17 @@ const DEFAULT_MODULE_FORMAT = 'getExport';
 export default async function bundleSource(
   startFilename,
   moduleFormat = DEFAULT_MODULE_FORMAT,
+  access
 ) {
-  const resolvedPath = path.resolve(startFilename);
+  const { rollup, resolvePlugin, pathResolve } = access || {
```

Comment:
As you've written this, you can either override all or none. Is this what you want? Or do you want to be able to override some and let the remainder default?

Reply:
I'm only interested in overriding all.

And the existing usage is to override none.